packages/lit-dev-content/site/blog/2023-10-10-lit-3.0.md

```diff
@@ -0,0 +1,262 @@
+
+Here are the biggest things Lit 3.0 changes:
+- IE11 is no longer supported.
+- Lit's npm modules are now published as ES2021.
+- APIs deprecated with the Lit 2.0 release have been removed.
+- SSR hydration support modules were moved to the `@lit-labs/ssr-client` package.
+- Decorator behavior has been unified between TypeScript experimental decorators and standard decorators.
+- Support was removed for Babel decorators version "2018-09"
+
+### New: standard decorators support
+
+The one new feature that we _did_ add to Lit 3.0 is support for the TC39 standard decorators specification to our _existing_ decorators.
+
+The new decorator spec has reached Stage 3 in TC39, meaning that browsers and compilers are now implementing them. This is a huge step for Lit! The arrival of standard decorators allows us to begin the process of moving to a decorator implementation that won't require a compiler to use.
+
+It's very important to us to make the upgrade path from experimental decorators as smooth as possible. To accomplish this we made our existing decorators support the standard spec and made them work with the new `accessor` keyword in experimental decorator mode.
+
+This way you can use Lit decorators with auto-accessors (using the `accessor` keyword) so that the same call site works with both settings:
+
+```ts
+class MyElement extends LitElement {
+  @property()
+  accessor myProperty = 'hello';
+}
+```
+
+Once all decorator call sites in your project use the `accessor` keyword, you can build with `experimentalDecorators` set to `true` or `false` without a change in behavior.
+
+But in order to make these hybrid decorators have consistent behavior in both modes we had to make a few minor breaking changes to our experimental decorators:
+
+- We now call `requestUpdate()` automatically for `@property` and `@state` decorated accessors where previously that was the setter's responsibility.
+- The value of an accessor is read on first render and used as the initial value for `changedProperties` and attribute reflection.
+- `@property` and `@state` must be used on the setter for hand-written accessors.
+
+Standard decorator mode requires [TypeScript 5.2](https://devblogs.microsoft.com/typescript/announcing-typescript-5-2/#decorator-metadata) or [Babel 7.23](https://babeljs.io/blog/2023/09/25/7.23.0) with the [`@babel/plugin-proposal-decorators`](https://babeljs.io/docs/babel-plugin-proposal-decorators) plugin using decorator version `"2023-05"`.
+
+### Upgrading
+
+The upgrade from Lit 2.0 should be seamless for the vast majority of users. You can usually upgrade your npm dependency version with:
+
+```sh
+> npm i lit@latest
+```
+
+You can find more details and how to handle the changes in the [Lit 3 upgrade guide](/docs/v3/releases/upgrade/).
+
+Detailed change logs can be found [on GitHub](https://github.com/lit/lit/releases?q=%22-pre.1%22&expanded=true).
```

Comment:
Changed search query to "3.0.0" rather than "-pre.1". We could even link to the `lit@3.0.0` tag but that'll have to be after it's published.
```suggestion
Detailed change logs can be found [on GitHub](https://github.com/lit/lit/releases?q=lit%403.0.0&expanded=true).
```